packages/datasource/src/nav/header/header.data-en.js

```diff
@@ -65,11 +65,11 @@ export const navItems = [
       },
       {
         title: 'User Group',
-        link: 'https://tug.tidb.io',
+        link: '/',
         items: [
           {
             title: 'Join TiDB User Group',
-            link: 'https://tug.tidb.io/people',
+            link: '/tug',
```

Comment:
It may break the existing logic of auto-mapping rules defined in https://github.com/pingcap/tug-website/blob/main/packages/datasource/src/nav/utils.js#L28-L47.

Please test your changes on preview, and ask @634750802 for his opinions as the mapping function was implemented by him.

Reply:
OK!